demos/python_demos/multi_camera_multi_person_tracking/multi_camera_multi_person_tracking.py

```diff
@@ -50,28 +75,39 @@ def __call__(self):
                 self.frames_queue.put(frames)
 
 
-def run(params, capture, detector, reid):
+def run(params, config, capture, detector, reid):
     win_name = 'Multi camera tracking'
-    config = {}
-    if len(params.config):
-        config = read_py_config(params.config)
+    frame_number = 0
+    avg_latency = AverageEstimator()
+    key = -1
+
+    if config['normalizer_config']['enabled']:
+        del config['normalizer_config']['enabled']
+        capture.add_transform(NormalizerCLAHE(**config['normalizer_config']))
```

Comment:
That's confusing when I see `if something: delete that something`. Why not 
```py
if config['normalizer_config']['enabled']:
    capture.add_transform(NormalizerCLAHE(config['normalizer_config']['clip_limit'],
        config['normalizer_config']['tile_size']))
```

Reply:
Fixed.